tests/test_preprints.py

```diff
@@ -293,6 +294,58 @@ def test_update_search(self, mock_update_preprint, preprint):
 
 class TestPreprintCreation:
 
+    @mock.patch('website.mails.send_mail')
+    def test_preprint_submitted_signal_non_moderated_provider(self, fake):
```

Comment:
This is technically valid, but you're shadowing the `fake` fixture in the `conftest.py` which would be imported here. It's more conventional to call this something like `mock_email`.